package.json

```diff
@@ -19,6 +19,8 @@
   },
   "homepage": "https://github.com/gemtechd/build-tests#readme",
   "author": "gemtechd",
-  "license": "ISC"
-
+  "license": "ISC",
+  "devDependencies": {
+    "nyc": "^17.0.0"
```

Comment:
what is this module?

Reply:
I installed it when I didn't manage with the tests, but in the end I managed.